### ISSUE TITLE
scalar with many spaces doesn't round trip [rt.cpan.org #7146]

### ISSUE THREAD

**Issue by ingydotnet** [ingydotnet/yaml-pm#40]
https://rt.cpan.org/Ticket/Display.html?id=7146

```
I found a problem round tripping a string containing a large number of spaces:

perl -MData::Dumper -MYAML -e '$orig = "A".(" "x200)."B"; $roundtrip = Load Dump $orig; print Dump($orig,$roundtrip); print Dumper([$orig,$roundtrip])' 

Pipe through cat -A to see the differences.

It seems like the loading of flow scalars isn't working right.

This is with version 0.35.
```

**Comment by perlpunk** [comment]
Created #172

**Comment by perlpunk** [comment]
merged #172
